glue/clients/util.py

```diff
@@ -4,11 +4,15 @@
 from functools import partial, wraps
 
 import numpy as np
+import pandas as pd
+from matplotlib.dates import date2num, AutoDateLocator, AutoDateFormatter
+import datetime
```

Comment:
nit: imports from the python standard lib go in the first block of imports (with functools above)